src/views/Home.jsx

```diff
@@ -25,28 +25,42 @@ export function Home({
 			setListToken(displayName, 'tcl-shopping-list-token');
 			navigate('/list');
 		} else {
-			setErrorMessage('Invalid token | The list does not exist');
+			setErrorMessage('Error! Invalid token: this list does not exist. :(');
```

Comment:
When you add 'Error!', it looks kinda scary like something went wrong, when it might be the case that the user just mistyped the token. 
Here are some tips for writing error messages https://uxplanet.org/how-to-write-good-error-messages-858e4551cd4

Reply:
I will take a look at it.
Thanks, Golfo.